Modules/Crm/Http/Controllers/ProjectsController.php

```
@@ -2,104 +2,60 @@
-        // TODO: Implement edit/form method if exists in original
+        if (request()->isMethod('post') && request()->post('btn_submit')) {
+            $validated = request()->validate(Project::validationRules());
+            if ($id) {
+                Project::findOrFail($id)->update($validated);
+            } else {
+                Project::create($validated);
+            }
+            return redirect()->route('projects.index')->with('alert_success', trans('record_successfully_saved'));
+        }
+
+        $project = $id ? Project::find($id) : new Project();
```

PHPMD (2.15.0)

Avoid using static access to class '\Modules\Crm\Entities\Project' in method 'form'. (undefined)

(StaticAccess)